okhttp/src/jsMain/kotlin/okhttp3/-JsPlatform.kt

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) 2022 Square, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package okhttp3
+
+import kotlin.coroutines.resumeWithException
+import kotlinx.coroutines.suspendCancellableCoroutine
+import okio.IOException
+
+suspend fun Call.jsExecuteAsync(): Response {
+  return suspendCancellableCoroutine { continuation ->
```

Comment:
This should be in nonJvmMain, but I can't reference the suspendCancellableCoroutine from that module.  It looks like it is defined in kotlix-coroutines-core-common, but only works for jvm and js.